.github/workflows/semantic-pr.yml

```diff
@@ -9,7 +9,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: "Check for semantic PR title"
+name: "PR Check Title"
```

Comment:
We should check again if this is just checking the PR title, or also the commit message. In case a commit message is also checked, the name should be adopted to make this clearer and not give a false impression.